content/10.diagnostics.md

```diff
@@ -58,14 +57,17 @@ False positives are a concern with this technique, as the guide RNA can recogniz
 
 Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
 
-#### Sierological Tests
+### 4.2 Sierological Tests
 
 Despite diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the real extent of the disease.
 In fact, they would not work on a recovered patient who defeated the virus.
 In this context, immunity tests are significantly more informative.
 In fact, they can help scientists figuring out why the disease has different course among patients, as well as what strategy might work to manage the spread of the infection.
```

Comment:
```suggestion
Additionally, they can help scientists to understand why the disease has a different course among patients, as well as what strategy might work to manage the spread of the infection.
```

Reply:
In fact is repeated two lines above. 

I said "to understand", alternatively I would go with "to figure out"